### PR TITLE
Fixed broken link && Changed http to https

### PR DESCRIPTION
**Pull request by Alaev**
### Description:

- Fixed broken link in readme
- Changed the following links from HTTP to HTTPS as it is supported by loopback and strongloop

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -78,23 +78,23 @@ The LoopBack community has created and supports a number of additional connector
 
 StrongLoop provides a number of example applications that illustrate various key LoopBack features. In some cases, they have accompanying step-by-step instructions (tutorials).
 
-See [loopback-example](https://github.com/strongloop/loopback-example) for details.
```
Comment (Alaev) [inline README.md]: This link was giving 404.

Comment (bajtos) [inline README.md]: Please change the link title too - remove `loopback-example`, use something like `examples at loopback.io` instead.

**Comment by bajtos** [comment]
@slnode ok to test

**Comment by bajtos** [comment]
> clahub — Not all contributors have signed the Contributor License Agreement

@Alaev before we can land your pull request, we need you to sign our CLA. Please open https://cla.strongloop.com/agreements/strongloop/loopback and follow the instructions there.

File: README.md (hunk 1 above)
Comment (Alaev) [inline README.md]: Ty for the comment. fixed.

**Comment by Alaev** [comment]
CLA signed.

**Comment by crandmck** [comment]
Since no one responded to my question in Slack, I'm going to land this, despite CI failing due to a few extra chars in the extended commit msg.

**Pull request merged**